src/Components/CaseStatusBar/CaseStatusBar.tsx

```diff
@@ -1,10 +1,71 @@
 import React from 'react';
 import { View, Text } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
 
-export default function CaseStatusBar() {
+import styles from './styles';
+import { colors } from '../../styles/colors';
+
+interface CaseStatusBarProps {
+  status: string;
+}
+
+export default function CaseStatusBar({ status }: CaseStatusBarProps) {
+  // stephanie this thing is so mf ugly where should i hide it
+  const getStatusColor = (status: string) => {
+    if (
+      status === 'In Progress' ||
+      status === 'New Case' ||
+      status === 'Settled' ||
+      status === 'Appeal' ||
+      status === 'Payment Processing' ||
+      status === 'Payment Distributed'
+    ) {
+      return {
+        background: {
+          backgroundColor: colors.lightGreen,
+          borderColor: colors.darkGreen,
+        },
+        text: { color: colors.darkGreen },
+      };
+    } else if (status === 'Pending') {
+      return {
+        background: {
+          backgroundColor: colors.lightYellow,
+          borderColor: colors.darkYellow,
+        },
+        text: { color: colors.darkYellow },
+      };
+    } else if (status === 'Action Required') {
+      return {
+        background: {
+          backgroundColor: colors.lightRed,
+          borderColor: colors.darkRed,
+        },
+        text: { color: colors.darkRed },
+      };
+    } else {
+      return {
+        background: {
+          backgroundColor: colors.lightGrey,
+          borderColor: colors.darkGrey,
+        },
+        text: { color: colors.darkGrey },
+      };
+    }
+  };
+  const statusColor = getStatusColor(status);
   return (
-    <View>
-      <Text>CaseStatusBar</Text>
-    </View>
+    <TouchableOpacity style={styles.caseStatusContainer}>
```

Comment:
Hmm I think it makes more sense to wrap the `TouchableOpacity` in a `View` rather than the other way around